mm2src/coins/qrc20.rs

```diff
@@ -1326,10 +1326,9 @@ async fn qrc20_withdraw(coin: Qrc20Coin, req: WithdrawRequest) -> WithdrawResult
         .map_to_mm(WithdrawError::InvalidAddress)?;
     let conf = &coin.utxo.conf;
     let is_p2pkh = to_addr.prefix == conf.pub_addr_prefix && to_addr.t_addr_prefix == conf.pub_t_addr_prefix;
-    let is_p2sh =
-        to_addr.prefix == conf.p2sh_addr_prefix && to_addr.t_addr_prefix == conf.p2sh_t_addr_prefix && conf.segwit;
-    if !is_p2pkh && !is_p2sh {
-        let error = "Expected either P2PKH or P2SH".to_owned();
+    let is_p2sh = to_addr.prefix == conf.p2sh_addr_prefix && to_addr.t_addr_prefix == conf.p2sh_t_addr_prefix;
+    if !is_p2pkh && is_p2sh {
```

Comment:
I think `is_p2sh` check is not required here. I can be simply `if !is_p2pkh`. Furthermore, I would also change the error message to provide a bit more context, e.g. `QRC20 can be sent to P2PKH addresses only`.